packages/components/src/components/form/BottomText.tsx

```diff
@@ -20,31 +20,40 @@ const slideDown = keyframes`
     }
 `;
 
-export const Container = styled.div<{ inputState?: InputState }>`
+export const Container = styled.div<Pick<BottomTextProps, 'inputState' | 'isDisabled'>>`
     display: flex;
     align-items: center;
     gap: ${spacingsPx.xxs};
     padding: ${spacingsPx.xs} ${spacingsPx.sm} 0 ${spacingsPx.sm};
     min-height: ${BOTTOM_TEXT_MIN_HEIGHT}px;
-    color: ${({ inputState, theme }) => getInputStateTextColor(inputState, theme)};
+    color: ${({ inputState, isDisabled, theme }) =>
+        isDisabled ? theme.textDisabled : getInputStateTextColor(inputState, theme)};
     ${typography.label}
     animation: ${slideDown} 0.18s ease-in-out forwards;
 `;
 
 interface BottomTextProps {
     inputState?: InputState;
+    isDisabled?: boolean;
     icon?: IconName;
     children: ReactNode;
 }
 
-export const BottomText = ({ inputState, icon = 'warningCircle', children }: BottomTextProps) => {
+export const BottomText = ({
+    inputState,
+    isDisabled,
```

Comment:
Not sure if needed or if it is even desirable. This way it will be easy to overlook some validation error which might block form submission. It could be some validation dependent on another, editable field.

Reply:
It is there for styling only and it's like that in Figma 🤔  I would say this state exists because `BottomText` !== `ErrorText` – anything can be there. And it's kinda hard to imagine why would you disable a field with an error state, or upon disabling the field would have to lose the error state? I don't think we have anything like this at the moment anyways.